lib/tutorials/en_US/validation.md

```diff
@@ -1,14 +1,45 @@
-## Validation
+# Validation
 
 _This tutorial is compatible with hapi v17_
 
+- [Overview](#overview)
+- [Joi](#joi)
+- [Input Validation](#input)
+    - [Path Parameters](#pathparams)
+    - [Query Parameters](#queryparams)
+    - [Payload Parameters](#payloadparams)
+    - [Headers](#headers)
+- [Output Validation](#output)
+    - [response.failAction](#failaction)
+    - [response.sample](#sample)
+    - [response.status](#status)
+    - [response.options](#options)
+- [Alternatives to Joi](#alternatives)
+
+
+
+
+## <a name="overview" /> Overview
+
 Validating data can be very helpful in making sure that your application is stable and secure. hapi allows this functionality by using the module [Joi](https://github.com/hapijs/joi), which allows you to create your validations with a simple and clear object syntax.
 
-## Input
+## <a name="joi" /> Joi
+
+[Joi](https://github.com/hapijs/joi) is an object schema description language and validator for JavaScript objects. Joi allows you to create blueprints or schemas for JavaScript objects to ensure validation of key information. Joi can validate both input and output data. To get started with `joi`, you must first install and add it as a dependency to your project:
+
+`npm install joi`
+
+Then, you must import it to your project:
+
+`const Joi = require('joi');`
+
+## <a name="input" /> Input Validation
+
+The first type of validation hapi can perform is input validation. This is defined in the `options` object on a route, and is able to validate headers, path parameters, query parameters, and payload data. Keep reading for examples of input validation with `joi`.
 
```

Comment:
I would add a mention about the fact that you can give to `route.options.validate` either JS object or Joi object as well. The latter allows you to set options for that validation. Here is how I would phrase it:

Notes: In the below examples, you'll see that we give JS object to route `options.validate`. Be aware that the `validate` option accepts either JS object or `joi` object for its properties. The latter allows you to set `joi` options for that particular schema. Here is a partial rewrite of the [Query Parameters](#queryparams) example:
```js
options: {
    validate: {
        query: Joi.object({
            limit: Joi.number().integer().min(1).max(100).default(10)
        }).options({ stripUnknown: true })
    }
}
```
Look [here](https://github.com/hapijs/joi/blob/v14.3.1/API.md#anyoptionsoptions) for details about such options.